### ISSUE TITLE
UriMapping not correctly translates to Blender.MapUri

### ISSUE THREAD

**Issue by warpech** [Starcounter/Home#141]
Until Starcounter 2.3.0, we advised apps to do the following mapping of the URIs:

```c#
UriMapping.Map("/launcher/menu", UriMapping.MappingUriPrefix + "/menu");
```

Now, we advise the following mapping of the URIs:

```c#
Blender.MapUri("/launcher/menu", "menu");
```

It is great that the two APIs are compatible. But one of them should use translation to remove `UriMapping.MappingUriPrefix + "/"` from the token, otherwise the old apps are not really compatible.

I suggest that before the lines https://github.com/Starcounter/level1/blob/1ede891584dfedd228bf19629f7c5e1b0e4732c4/src/Starcounter/Apps/OntologyAndOrdinaryMapping.cs#L53-L58

We add:

```c#
mapUri = mapUri.Replace(UriMapping.MappingUriPrefix + "/", "");
```

@alemoi WDYT?

@mmnosek recently had a problem with this. I believe @un-tone too.

**Comment by alemoi** [comment]
Sure, already added this change. But I also added Obsolete (as we discussed, once Blender is stable enough) message like this everywhere: 
```
[Obsolete("UriMapping.OntologyMap is no longer maintained. Please use Blender API instead: https://docs.starcounter.io/guides/blending/blending/")]
```
Is it ok?

**Comment by warpech** [comment]
LGTM

**Comment by warpech** [comment]
"UriMapping.OntologyMap is no longer maintained" gives rather a negative impression.

"UriMapping.OntologyMap is deprecated and will be removed in the next version of Starcounter" would be better.

Other than that, good idea to add this message!

**Comment by alemoi** [comment]
Merged and pushed into v 2.3.1 and 2.4.
